README.md

```diff
@@ -50,7 +53,7 @@ You can then pass this POJO to whatever CSS-in-JS framework you're using, i.e. w
 
 function MyReactComponent(props: MyProps) {
   // Use emotion's css prop
-  return <div css={Css.mx2.black.$}>content</div>
+  return <div css={Css.mx2.black.$}>content</div>;
```

Comment:
All changes after this line are auto-formatted. Let me know if you'd like them to be changed back.